pkg/apis/extensions/v1beta1/types.go

```diff
@@ -793,6 +793,9 @@ type ExportOptions struct {
 type ListOptions struct {
 	unversioned.TypeMeta `json:",inline"`
 
+	// Labels for reverse lookup
+	// Defaults to match any selector
+	Labels string `json:"labels,omitempty"`
```

Comment:
Just `labels` may be confusing. We actually used that very early on rather than `labelSelector`. `labelsSelected` would be more accurate, but too similar to `labelSelector`. `selectedLabels` sounds like a chosen subset, which isn't the right connotation. Since we use "match" in the newer selector, `matchedLabels` could work.

However, the main thing we need to consider is whether we expect to have any cases where there would be multiple independent selectors in any given resource and, if so, how to handle that. I suppose we could add another parameter to handle that case. 

Our controllers (e.g., DaemonSet) all have both a pod selector and a node selector (in the pod template). For now, I'd expect we'd apply the pod selector only.